int64[4] scan
---
string result
